docs/help/contributor/README.md

```diff
@@ -5,6 +5,7 @@
 
 ### Getting started
 
+- [Code of Conduct](https://knative.dev/community/contributing/code-of-conduct)
```

Comment:
The CoC is linked inside `becoming-a-contributor.md` (same directory). Is that OK? Do we want to keep bother /  remove one?

Reply:
That's a fair point, but I believe the concern of the issue is to give the Code of Conduct more visibility so I'm okay linking to it in both places.